Products/CMFPlone/CatalogTool.py

```diff
@@ -379,6 +379,51 @@ def getCounter(self):
         processQueue()
         return self._counter is not None and self._counter() or 0
 
+    @security.private
+    def allow_inactive(self, query_kw):
+        """Check, if the user is allowed to see inactive content.
+        First, check if the user is allowed to see inactive content site-wide.
+        Second, if there is a 'path' key in the query, check if the user is
+        allowed to see inactive content for these paths.
+        Conservative check: as soon as one path is disallowed, return False.
+        If a path cannot be traversed, ignore it.
+        """
+        allow_inactive = _checkPermission(AccessInactivePortalContent, self)
+        if allow_inactive:
+            return True
+
+        paths = query_kw.get('path', False)
+        if not paths:
+            return []
```

Comment:
That's a leftover from refactoring this method a couple of times... I'll change to ``False``.